po/bn.po

```diff
@@ -251,431 +251,431 @@ msgstr "দিন 2: সকাল"
 
 #: src/SUMMARY.md:76
 msgid "Structs"
-msgstr "কাঠামো"
+msgstr "Structs"
 
 #: src/SUMMARY.md:77
 msgid "Tuple Structs"
 msgstr "Tuple কাঠামো"
 
 #: src/SUMMARY.md:78
 msgid "Field Shorthand Syntax"
-msgstr ""
+msgstr "Field Shorthand Syntax"
 
 #: src/SUMMARY.md:79
 msgid "Enums"
-msgstr ""
+msgstr "Enums"
 
 #: src/SUMMARY.md:80
 msgid "Variant Payloads"
-msgstr ""
+msgstr "ভিন্ন Payloads"
 
 #: src/SUMMARY.md:81
 msgid "Enum Sizes"
-msgstr ""
+msgstr "Enum এর মাপ"
 
 #: src/SUMMARY.md:83
 msgid "Method Receiver"
-msgstr ""
+msgstr "Receiver পদ্ধতি"
 
 #: src/SUMMARY.md:84 src/SUMMARY.md:159 src/SUMMARY.md:272
 msgid "Example"
-msgstr ""
+msgstr "উদাহরণ"
 
 #: src/SUMMARY.md:85
 msgid "Pattern Matching"
-msgstr ""
+msgstr "প্যাটার্ন মিলানো"
 
 #: src/SUMMARY.md:86
 msgid "Destructuring Enums"
-msgstr ""
+msgstr "Enums ধ্বংস করা"
```

Comment:
Destructuring doesn't mean "ধ্বংস করা"| ডিস্ট্রাকচার is probably the appropriate term, as we don't have a well esablished form of this term.